platform/src/main/java/org/hillview/main/DemoDataCleaner.java

```diff
@@ -85,6 +86,27 @@ public static void main(String[] args) throws IOException {
                         owriter.writeTable(p);
                     }
 
+                    final String parquetFileName = end.replace(".orc", ".parquet");
```

Comment:
Can you make this conditional on a command -line flag?

Reply:
Yes, I'll add a flag

Reply:
Added a flag `-p` to `demo-data-cleaner.sh` to control the generation of parquet files. Also changed the flag for delta table from `-g` to `-d`.